packages/grafana-ui/src/components/ColorPicker/SeriesColorPickerPopover.tsx

```diff
@@ -19,20 +19,22 @@ export const SeriesColorPickerPopover: FunctionComponent<SeriesColorPickerPopove
       customPickers={{
         yaxis: {
           name: 'Y-Axis',
-          tabComponent: () => (
-            <Switch
-              key="yaxisSwitch"
-              label="Use right y-axis"
-              className="ColorPicker__axisSwitch"
-              labelClass="ColorPicker__axisSwitchLabel"
-              checked={yaxis === 2}
-              onChange={() => {
-                if (onToggleAxis) {
-                  onToggleAxis();
-                }
-              }}
-            />
-          ),
+          tabComponent() {
```

Comment:
Interesting, is this related to the rules enabled in this PR?

Reply:
Yes, this is related to the `react/display-name` rule.

Since arrow function is  anonymous, the declared component will not have the displayName. So I changed this to using shorthand method function definition to creates a named function.

Reply:
Perfect, thanks for the explanation:)